README.md

```diff
@@ -104,10 +112,10 @@ Don't forget to logout (or Ctrl-D) or you may find yourself heading all the way
 $ multipass exec dancing-chipmunk -- lsb_release -a
 
 No LSB modules are available.
-Distributor ID:  Ubuntu
-Description:     Ubuntu 18.04.1 LTS
-Release:         18.04
-Codename:        bionic
+Distributor ID:	 Ubuntu
```

Comment:
can we make the 2nd column left aligned like the text before?

Reply:
OK I'll check alignment here!

Reply:
Now, it looks good.